### PR TITLE
Fixes Duke Nukem announcer having 10 weight

### PR DESCRIPTION
**Pull request by cowbot92**
# Document the changes in your pull request

Fixes the duke nukem announcer having a base of 10 weight

# Why is this good for the game?

Something was wrong

# Testing

I wouldnt make this if i didnt open a private server 10 times to hear duke (editor's note my fix still worked somehow, not sure why)

# Changelog

:cl:  

tweak:  tweaks how often duke nukem talks to us

/:cl:

### CODE REVIEW

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 /datum/station_trait/announcement_duke
 	name = "Announcement Duke"
 	trait_type = STATION_TRAIT_NEUTRAL
-	weight = 0.05
+	weight = 0.2
```
Comment (Bibby0110) [inline code/datums/station_traits/neutral_traits.dm]:
```suggestion
	weight = 0.05
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Change /datum/station_trait/announcement_duket to /datum/station_trait/announcement_duke on line 75

**Comment by Yarinoi** [comment]
whoops
but frankly, is this REALLY a flaw that needs to be corrected?

File: code/datums/station_traits/neutral_traits.dm (hunk 1 above)
Comment (cowbot92) [reply code/datums/station_traits/neutral_traits.dm]: Changes made

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**